src/library_manager/lib_manager.c

```diff
@@ -578,6 +647,13 @@ static int lib_manager_store_library(struct lib_manager_dma_ext *dma_ext,
 	tr_dbg(&lib_manager_tr, "lib_manager_store_library(): pointer: %p",
 	       (__sparse_force void *)library_base_address);
 
+#if CONFIG_LIBRARY_AUTH_SUPPORT
+	ret = lib_manager_auth_proc(man_buffer,
+				    MAN_MAX_SIZE_V1_8, AUTH_PHASE_FIRST);
+	if (ret < 0)
```

Comment:
need to free memory - check like 665 below

Reply:
fixed